admin/data/sql/migrations/20200210153633.undo.update-sp-create-checks-pt-1.sql

```diff
@@ -0,0 +1,135 @@
+CREATE OR ALTER PROCEDURE [mtc_admin].[spCreateChecks]
+@TVP [mtc_admin].[CheckTableType] READONLY
+AS
+SET ANSI_NULLS ON
+SET QUOTED_IDENTIFIER ON
+SET CONCAT_NULL_YIELDS_NULL ON
+SET ANSI_WARNINGS ON
+SET ANSI_PADDING ON
+    -- Connection pooling is enabled - make sure we are not in an existing transaction
+    IF @@TRANCOUNT <> 0  -- Rollback old transactions before starting another
+        ROLLBACK TRANSACTION
+
+BEGIN TRY
+    BEGIN TRANSACTION
+
+        DECLARE checkArgsList CURSOR
+            FOR SELECT pupil_id, checkForm_id, checkWindow_id, isLiveCheck, pinExpiresAt, school_id
+                FROM @TVP
+            FOR READ ONLY
+
+        DECLARE @pupilId int
+        DECLARE @checkFormId int
+        DECLARE @checkWindowId int
+        DECLARE @isLiveCheck bit
+        DECLARE @pinExpiresAt datetimeoffset
+        DECLARE @schoolId int
+        DEClARE @checkId int
+        DECLARE @output TABLE (id int);
+        DECLARE @isRestart bit;
+        DECLARE @pupilRestartId int;
+
+        OPEN checkArgsList
+        FETCH checkArgsList INTO @pupilId, @checkFormId, @checkWindowId, @isLiveCheck, @pinExpiresAt, @schoolId
+        WHILE (@@FETCH_STATUS = 0) BEGIN
+
+            -- Create the check
+            INSERT INTO [mtc_admin].[check]
+            (pupil_id, checkForm_id, checkWindow_id, isLiveCheck)
+            VALUES (@pupilId, @checkFormId, @checkWindowId, @isLiveCheck);
+
+            -- Get the check.id we just inserted
+            SET @checkId = SCOPE_IDENTITY();
+
+            -- Assign a pin to the check
+            INSERT INTO [mtc_admin].[checkPin] (school_id, check_id, pinExpiresAt, pin_id)
```

Comment:
This appears to be a very good reason to leave it alone - it should be pinExpiresAtUtc according to the migration above.

Just leave it - it's a minor change that has lots of knock-on effects.